packages/gatsby-transformer-xml/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -0,0 +1,301 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Process XML nodes correctly correctly creates nodes from XML 1`] = `
+Array [
+  Array [
+    Object {
+      "attributes": Object {
+        "id": "bk101",
+      },
+      "children": Array [
+        Object {
+          "attributes": Object {},
+          "children": Array [],
+          "content": "Gambardella, Matthew",
+          "name": "author",
+        },
+        Object {
+          "attributes": Object {},
+          "children": Array [],
+          "content": "XML Developer's Guide",
+          "name": "title",
+        },
+        Object {
+          "attributes": Object {},
+          "children": Array [],
+          "content": "Computer",
+          "name": "genre",
+        },
+        Object {
+          "attributes": Object {},
+          "children": Array [],
+          "content": "44.95",
+          "name": "price",
+        },
+        Object {
+          "attributes": Object {},
+          "children": Array [],
+          "content": "2000-10-01",
+          "name": "publish_date",
+        },
+        Object {
+          "attributes": Object {},
+          "children": Array [],
+          "content": "An in-depth look at creating applications 
+            with XML.",
+          "name": "description",
+        },
+      ],
+      "content": "",
+      "id": "bk101",
+      "internal": Object {
+        "contentDigest": "446732570969e52599dded1ba4941b65",
+        "type": "UndefinedXml",
```

Comment:
@KyleAMathews I just found the same issue in the [`gatsby-transform-json`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-json/src/__tests__/__snapshots__/gatsby-node.js.snap#L26) should it be `node.internal.name` instead of `node.name` ?